Libraries/Components/WebView/WebView.android.js

```diff
@@ -160,6 +160,20 @@ class WebView extends React.Component {
      * @platform android
      */
     allowUniversalAccessFromFileURLs: PropTypes.bool,
+
+    /**
+     * Rules that will be used to block the webview navigation
+     */
+    navigationBlockingPolicies: PropTypes.arrayOf(PropTypes.shape({
+      currentURL: PropTypes.string,
+      url: PropTypes.string,
```

Comment:
Seems from the blocking logic that `currentURL` is the "source" URL, i.e. the URL we're going from. Whilst `url` is the "request" URL, i.e. the URL we're going to. Maybe the naming should reflect that a bit better.

Also both of them are matched as regex, so the docs should reflect that.